secrets/awsssm/aws_ssm.go

```diff
@@ -97,12 +104,15 @@ func (a *AwsSsmManager) GetSecret(name string) ([]byte, error) {
 
 // SetSecret saves a secret to AWS SSM
 func (a *AwsSsmManager) SetSecret(name string, value []byte) error {
-	if _, err := a.client.PutParameter(&ssm.PutParameterInput{
-		Name:      aws.String(a.constructSecretPath(name)),
-		Value:     aws.String(string(value)),
-		Type:      aws.String(ssm.ParameterTypeSecureString),
-		Overwrite: aws.Bool(false),
-	}); err != nil {
+	if _, err := a.client.PutParameter(
+		context.TODO(),
```

Comment:
Should there be any timeout?

Reply:
no need it, aws sdk exist [retries timeout](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/retries-timeouts/)